lib/perl/Genome/Model/Tools/Sx/Split/ByNs.pm

```diff
@@ -0,0 +1,56 @@
+package Genome::Model::Tools::Sx::Split::ByNs;
+
+use strict;
+use warnings;
+
+use Genome;
+
+class Genome::Model::Tools::Sx::Split::ByNs {
+    is  => 'Genome::Model::Tools::Sx::Base',
+    has => {
+        number_of_ns => {
+            is => 'Integer',
+            doc => 'Number of Ns to split sequence.',
+        },
+    },
+};
+
+sub help_brief { 'Split sequences by Ns' }
+
+sub help_detail { 'Split sequences by a set number of Ns. Does not support splitting qualites.' }
```

Comment:
`qualites` would benefit from an additional `i`.  (Would it make sense to say it only supports splitting on sequence instead?  There are a great many things in the universe that it doesn't support!)